README.md

```diff
@@ -675,6 +848,9 @@ it('works', async () => {
     assert(fs.existsSync('lib/add.js'))
     await testCommand.handler({ })
     assert(fs.existsSync('coverage/lcov.info'))
+    await lintCommand.handler({ _: ['fix'] })
+    assert(fs.existsSync('README.md'))
+    assert(fs.readFileSync('README.md', 'utf8') === example)
```

Comment:
…then on the assertion, you can make it match against `example.replace('a+b', 'a + b')` to verify that the code has indeed been fixed.